Userland/Libraries/LibVideo/VP9/Decoder.h

```diff
@@ -10,15 +10,19 @@
 #include "LookupTables.h"
 #include "ProbabilityTables.h"
 #include "SyntaxElementCounter.h"
+#include "TreeParser.h"
 #include <AK/ByteBuffer.h>
 #include <AK/OwnPtr.h>
 
 namespace Video::VP9 {
 
 class Decoder {
+    friend class TreeParser;
+
 public:
     Decoder();
-    bool parse_frame(const ByteBuffer&);
+    virtual ~Decoder();
```

Comment:
Why is this `virtual`? Do you plan to inherit from that class in a further PR?

Reply:
I'm actually not sure why this was changed, I have no recollection of writing this (some of these changes are months old). Thanks for the tip-off, I'll look into it!